ale_linters/ruby/solargraph.vim

```diff
@@ -5,6 +5,7 @@
 " Description: updated to use stdio
 
 call ale#Set('ruby_solargraph_executable', 'solargraph')
+call ale#Set('ruby_solargraph_init_options', {})
```

Comment:
Rename that to just `_options` instead of `_init_options`, and I think this should be good.